code/noRAM/app/src/main/java/com/example/noram/AttendeeAnnouncementsActivity.java

```diff
@@ -0,0 +1,21 @@
+package com.example.noram;
+
+import android.os.Bundle;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.example.noram.model.Event;
+
+public class AttendeeAnnouncementsActivity extends AppCompatActivity {
+    private static final String eventIDLabel = "eventID";
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState){
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.attendee_announcements);
+
+        // retrieve corresponding event in database
+        int eventID = getIntent().getIntExtra(eventIDLabel,0);

```

Comment:
Whats this getIntent we use storage refs for the db

Reply:
The getIntent is to get the event whose notifications we want to display. 

AttendeeAnnouncementsActivity gets called when someone clicks on an Event's notifications, where the Event activity will swap with Announcements activity while specifying which event notifications we want to display with the Intent

Reply:
ahhhh i see